buildid/examples/taskrun.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
```

Comment:
I believe  `TaskRun` has already been covered as part of `buildid/tests` 
IMO having it again in examples folder will be useful if we are showing something different scenario like `buildid/examples/pipeline.yaml`

Let me know what you think on this

Reply:
I debated that too and decided that I would keep the content in the tests different than the one in examples as over time I might have more tests added to the same test configuration without necessarily updating the example (IMO the example should be fairly simple for anyone to understand)